pennylane_qiskit/converter.py

```diff
@@ -328,56 +327,62 @@ def _function(*args, params: dict = None, wires: list = None, **kwargs):
         wire_map = map_wires(qc_wires, wires)
 
         # Stores the measurements encountered in the circuit
-        mid_circ_meas, terminal_meas = [], []
+        terminal_meas = []
+        mid_circ_meas, mid_circ_regs = [], {}
 
         # Processing the dictionary of parameters passed
-        for idx, (op, qargs, _) in enumerate(qc.data):
-            # the new Singleton classes have different names than the objects they represent, but base_class.__name__ still matches
-            instruction_name = getattr(op, "base_class", op.__class__).__name__
-
-            operation_wires = [wire_map[hash(qubit)] for qubit in qargs]
-
+        for idx, (ops, qargs, cargs) in enumerate(qc.data):
```

Comment:
Is `ops` multiple operations or a single operation?

Reply:
It's a single operation. Changed it to `instruction`.